### PR TITLE
style: add PHP style checking

### PR DESCRIPTION
**Pull request by wreality**
This PR implements phpcs for style checking & fixes standards violations that already existed for PHP.  Javascript style changes will come in a separate PR.

I used CakePHP's coding standard as a base as its decently opinionated while still being reasonable.  Inside `/phpcs.xml` you'll find that some of the rules has been disabled since they were CakePHP specific.

Importantly, check and make sure that we all agree with the style choices that were made.  Unfortunately there isn't a way to generate a list of sniffs enforced... I looked... I know that would be useful.  Docs have also been updated with instructions on how to run tests locally: https://deploy-preview-137--eloquent-galileo-3be94a.netlify.app/contributing/testing.html#code-style-linting

Review Criteria:
- [x] Team agrees with the style choices
- [x] Automated style checks pass

### CODE REVIEW

**Comment by gmeben** [comment]
I'll review this first thing in the morning

**Review by gmeben — COMMENTED** [review]
I encounter this error when attempting to lint the PHP: `Script vendor/bin/phpcs handling the lint event returned with error code 127` which seems to indicate it's not picking up on the new dependency from the repo.

**Review by gmeben — APPROVED** [review]
I was in the wrong directory (`backend`) when I attempted to install the new dependency with `lando composer install`. This now runs successfully for me.

File: docs/contributing/testing.md [hunk 1]
```diff
@@ -113,3 +113,35 @@ It is possible to run the interactive test runner, although it does require some
...
+### PHP
+
+From the root of the project run:
+```sh
+lando composer lint
+```
+
+#### Integration into VScode
+
+The [PHP Sniffer](https://marketplace.visualstudio.com/items?itemName=wongjn.php-sniffer) extension should pick up our phpcs config automatically.  You will need to have PHP installed locally, however.
```
[the remark this answers is not in the document]
Comment (wreality) [reply docs/contributing/testing.md:130]: I installed v8.0 just to be consistent, but it _shouldn't_ matter....?

**Pull request merged**